tree/ntuple/test/ntuple_merger.cxx

```diff
@@ -1949,7 +1949,7 @@ TEST(RNTupleMerger, MergeAsymmetric1TFileMerger)
          fileMerger.SetMergeOptions(TString("rntuple.MergingMode=Strict"));
          CheckDiagsRAII diags;
          diags.requiredDiag(kError, "TFileMerger::Merge", "error during merge", false);
-         diags.requiredDiag(kError, "RNTuple::Merge", "missing the following field", false);
+         diags.requiredDiag(kError, "ROOT.NTuple", "missing the following field", false);
```

Comment:
I fully support the move from `TError` to `RLogger`.  It seems we need to add more details to the messages though.  The old messages are, i.e. something like:
```
"Error in RNTuple::Merge: missing the following field ..."
```
has more information (namely the activity that failed) than the new message, something like (assuming I am reading correctly):
```
Error ROOT.NTuple: missing the following field ..."
```

Unless the user provides additional context, this message could 'a priori' come from any operation that expects/needs a field.  

In the output of a large framework run, this might make the difference between knowing immediately where the problem is and not knowing until having instrumented the (user) code and/or put a breakpoint in the error logger

Reply:
That makes sense. In this case, would it be fine to use a different log channel, like `ROOT.NTuple.Merge`?

Reply:
I think so.